File: templates/python/logistics_network_state_map.py
"""logistics_network_state_map: 物流与网络状态热力图（heatmap 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='logistics and network analysis: state heatmap'):
    return make_template_figure('heatmap', seed=3403, title=title, domain='logistics and network analysis', topic='state heatmap')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/logistics_network_state_map.m
function fig = logistics_network_state_map()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('heatmap', 3403, 'logistics and network analysis: state heatmap', 'logistics and network analysis', 'state heatmap');
end
